src/sagemaker/jumpstart/notebook_utils.py

```diff
@@ -134,6 +138,7 @@ def extract_framework_task_model(model_id: str) -> Tuple[str, str, str]:
 def list_jumpstart_tasks(  # pylint: disable=redefined-builtin
     filter: Union[Operator, str] = Constant(BooleanValues.TRUE),
     region: str = JUMPSTART_DEFAULT_REGION_NAME,
+    sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
```

Comment:
sanity-check: is that okay to use this default in the arg list instead of in the method body? As in, it won't throw in some regions, correct?

Reply:
we've had this problem before, but this error gets suppresed cause of this try-catch: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/jumpstart/constants.py#L228-L237